src/ORM/Table.php

```diff
@@ -1531,7 +1531,7 @@ public function get($primaryKey, array $options = []): EntityInterface
         if ($cacheConfig) {
             if (!$cacheKey) {
                 $cacheKey = sprintf(
-                    'get:%s.%s%s',
+                    'get-%s.%s%s',
```

Comment:
```suggestion
                    'get-%s-%s-%s',
```
Why not just make it all dashes? Does json_encode always give a value that works for all caches?

Reply:
> Does json_encode always give a value that works for all caches?

Yes, the contents of the cache entry are fine, keys can't use filesystem reserved characters though.